thriftclient/client_pool.go

```diff
@@ -217,9 +217,9 @@ func NewWrappedTClientFactory(base TClientFactory, middlewares ...Middleware) TC
 // convenience struct for passing around the different factories needed to
 // create a Client.
 type factories struct {
-	client  ClientFactory
-	tclient TClientFactory
-	proto   thrift.TProtocolFactory
+	Client   ClientFactory
+	TClient  TClientFactory
+	Protocol thrift.TProtocolFactory
```

Comment:
This is an unexported struct so making the fields exported won't have any effect, but if your intention is to make it more obvious to tell fields and variables apart then this is fine :)

Reply:
Yep! It’s just to improve readability